$MirOS: ports/games/frozen-bubble/patches/patch-Makefile,v 1.1 2005/06/24 17:41:33 tg Exp $
$NetBSD: patch-ab,v 1.3 2003/04/16 22:04:32 salo Exp $
--- Makefile.orig	Thu Oct  5 19:38:47 2006
+++ Makefile	Sat Nov  4 03:05:23 2006
@@ -1,10 +1,9 @@
 DIRS = c_stuff po server
 
-PREFIX = /usr/local
 DATADIR = $(PREFIX)/share
 LIBDIR = $(PREFIX)/lib
 BINDIR = $(PREFIX)/bin
-MANDIR = $(DATADIR)/man
+MANDIR = $(PREFIX)/man
 
 all: prepare dirs
 
@@ -27,12 +26,10 @@ install: $(ALL)
 	@for n in $(DIRS); do \
 		(cd $$n; $(MAKE) install) \
 	done
-	install -d $(DESTDIR)$(BINDIR)
-	install frozen-bubble frozen-bubble-editor $(DESTDIR)$(BINDIR)
-	install -d $(DESTDIR)$(DATADIR)/frozen-bubble
-	cp -a gfx snd data $(DESTDIR)$(DATADIR)/frozen-bubble
-	install -d $(DESTDIR)$(MANDIR)/man6
-	install doc/*.6 $(DESTDIR)$(MANDIR)/man6
+	${BSD_INSTALL_SCRIPT} frozen-bubble frozen-bubble-editor ${DESTDIR}$(BINDIR)/
+	${BSD_INSTALL_DATA_DIR} ${DESTDIR}$(DATADIR)/frozen-bubble
+	cp -R gfx snd data ${DESTDIR}$(DATADIR)/frozen-bubble
+	${BSD_INSTALL_MAN} doc/*.6 ${DESTDIR}$(MANDIR)/man6/
 
 clean: 
 	@for n in $(DIRS); do \
